### ISSUE TITLE
Installation conflicts on ubuntu

### ISSUE THREAD

**Issue by Abdullah2993** [dotnet/cli#2457]
```
abdullah@vm:~$ sudo dpkg -i Desktop/dotnet-host-ubuntu-x64.latest.deb
dpkg: regarding .../dotnet-host-ubuntu-x64.latest.deb containing dotnet-host:
 dotnet-host conflicts with dotnet-nightly
  dotnet-nightly (version 1.0.1.001698-1) is present and installed.

dpkg: error processing archive Desktop/dotnet-host-ubuntu-x64.latest.deb (--install):
 conflicting packages - not installing dotnet-host
Errors were encountered while processing:
 Desktop/dotnet-host-ubuntu-x64.latest.deb
```

after uninstalling `dotnet-nightly` there is a conflict with xamarin

```
abdullah@vm:~$ sudo dpkg -i Desktop/dotnet-host-ubuntu-x64.latest.deb
(Reading database ... 152071 files and directories currently installed.)
Preparing to unpack .../dotnet-host-ubuntu-x64.latest.deb ...
Unpacking dotnet-host (1.0.0-rc2-002374-1) ...
dpkg: error processing archive Desktop/dotnet-host-ubuntu-x64.latest.deb (--install):
 trying to overwrite '/usr/share/dotnet', which is also in package mono-runtime-common 4.2.3.4-0xamarin2
dpkg-deb: error: subprocess paste was killed by signal (Broken pipe)
Errors were encountered while processing:
 Desktop/dotnet-host-ubuntu-x64.latest.deb
```

**Comment by shahid-pk** [comment]
Is it this issue https://github.com/dotnet/cli/issues/1162 resurfaced/regression ? I have installed dotnet-dev-rc versions from the debian feed and they don't have this problem.

**Comment by blackdwarf** [comment]
@brthor have you seen this?

**Comment by brthor** [comment]
the conflict with `dotnet-nightly` is by design.

The mono conflict was resolved in #1264

I've self assigned to ensure there wasn't a regression.
